sdk/keyvault/Azure.Security.KeyVault.Certificates/tests/CertificateClientLiveTests.cs

```diff
@@ -557,14 +558,175 @@ public async Task ValidateMergeCertificate()
             CollectionAssert.AreEqual(mergedServerCertificate.Cer, completedServerCertificate.Cer);
         }
 
-        // Backup Restore
-        // GetCertificates
-        // GetCertificatsIncludePending
-        // GetCertificateVersions
-        // GetDeletedCertificates
-        // GetUpdatePolicy
-        // IssuerCrud
-        // ContactsCrud
+        [Test]
+        public async Task VerifyGetIssuer()
+        {
+            string issuerName = Recording.GenerateId();
+
+            string providerName = "SSLAdmin";
+
+            CertificateIssuer issuer = new CertificateIssuer(issuerName, providerName);
+
+            RegisterForCleanupIssuer(issuerName);
+
+            await Client.CreateIssuerAsync(issuer);
+
+            CertificateIssuer getIssuer = await Client.GetIssuerAsync(issuerName);
+
+            Assert.NotNull(getIssuer);
+            Assert.NotNull(getIssuer.Id);
+            Assert.AreEqual(issuer.Provider, getIssuer.Provider);
+            //Assert.AreEqual(issuer.Name, getIssuer.Name);
+        }
+
+        [Test]
+        public async Task VerifyUpdateIssuer()
+        {
+            string issuerName = Recording.GenerateId();
+
+            string providerName = "SSLAdmin";
+
+            CertificateIssuer issuer = new CertificateIssuer(issuerName, providerName);
+
+            RegisterForCleanupIssuer(issuerName);
+
+            await Client.CreateIssuerAsync(issuer);
+
+            string updateProvider = "OneCert";
+            issuer = new CertificateIssuer(issuerName, updateProvider);
+            Assert.NotNull(issuer);
+
+            CertificateIssuer updateIssuer = await Client.UpdateIssuerAsync(issuer);
+
+            Assert.NotNull(updateIssuer);
+            Assert.NotNull(updateIssuer.UpdatedOn);
+            Assert.AreEqual(updateProvider, updateIssuer.Provider);
+        }
+
+        [Test]
+        public async Task VerifyGetPropertiesOfIssuersAsync()
+        {
+            string issuerName = Recording.GenerateId();
+            string issuerName1 = Recording.GenerateId();
+
+            string providerName = "SSLAdmin";
+            string providerName1 = "OneCert";
+
+            CertificateIssuer issuer = new CertificateIssuer(issuerName, providerName);
+            CertificateIssuer issuer1 = new CertificateIssuer(issuerName1, providerName1);
+
+            RegisterForCleanupIssuer(issuerName);
+            RegisterForCleanupIssuer(issuerName1);
+
+            await Client.CreateIssuerAsync(issuer);
+            await Client.CreateIssuerAsync(issuer1);
+
+            List<IssuerProperties> issuerProperties = await Client.GetPropertiesOfIssuersAsync().ToEnumerableAsync();
+
```

Comment:
### Issue2.

Description:
Use method GetPropertiesOfIssuersAsync() get different Provider Name for CertificateIssuer created in Portal and created by SDK.

Reproduce Steps:

1. Create CertificateIssuer.
    a. Create by Portal
         Navigate to Certificate -> Certificate Authorties, create a issuer named "test" with provider "SSLAdmin"
     b. Create by code
         ```
         var issuer = new CertificateIssuer("test","SSLAdmin");
         var get = await client.CreateIssuerAsync(issuer);
         ```   
2. Get the provider name
     a. The return value of Provider name pre created by portal
         Porvider = "SslAdmin";
     b. The return value of Provider name pre created by code
         Provider = "SSLAdmin";

Reply:
The name is not required to be case sensitive. Perhaps use the `StringAssert` class instead and use a case-insensitive comparison, but it's probably not necessary anyway since this test only has to be repeatable. We're asserting that what the test set up is what we got back. In practice, either name casing would work and no one should assert that.